### PR TITLE
Add german language support

### LINKED ISSUE #43: Add German translation

### PR DESCRIPTION
**Pull request by rdwz**
resolves #43

### CODE REVIEW

**Comment by helmasaur** [comment]
Thank you for your work! I'll review the code this week 😊.

**Comment by helmasaur** [comment]
I had lots of things to do recently, sorry for the delay. I'm looking through it right now!

File: locales/de/zodiac.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Widder",
+		"element": 0,
+		"stone": "Blutstein"
+	},
+	"taurus": {
+		"name": "Stier",
+		"element": 1,
+		"stone": "Saphir "
```
Comment (helmasaur) [inline locales/de/zodiac.json:10]: Typo: space

Comment (helmasaur) [inline locales/de/zodiac.json:10]: Typo: space

File: locales/de/zodiac.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	},
+	"capricorn": {
+		"name": "Steinbock",
+		"element": 1,
+		"stone": "Rubin"
+	},
+	"aquarius": {
+		"name": "Wassermann",
+		"element": 2,
+		"stone": "Granat"
```
Comment (helmasaur) [inline locales/de/zodiac.json:55]: Is it the right translation?

- https://de.wikipedia.org/wiki/Granatgruppe
- https://en.wikipedia.org/wiki/Garnet

File: locales/de/zodiac.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Widder",
+		"element": 0,
+		"stone": "Blutstein"
```
Comment (helmasaur) [inline locales/de/zodiac.json:5]: Is it the right translation?

- https://de.wikipedia.org/wiki/Heliotrop_(Mineral)
- https://en.wikipedia.org/wiki/Heliotrope_(mineral)

**Review by helmasaur — CHANGES_REQUESTED** [review]

**Comment by helmasaur** [comment]
Hello! Are you still up to make the modifications?

File: locales/de/zodiac.json (hunk 2 above)
Comment (VitoMars) [reply locales/de/zodiac.json:55]: "Granat" is correct.

Literally "Granatgruppe" means "Group of Garnet" which indicates all the compound of minerals, the stone in German is commonly called "Granat".
I don't know why here: https://de.wikipedia.org/wiki/Granatgruppe abbreviates it to "Granate" which literally means "Grenade", but I'm pretty sure the stone is called "Granat"

File: locales/de/zodiac.json (hunk 3 above)
Comment (VitoMars) [reply locales/de/zodiac.json:5]: "Heliotrop" is the correct word.  

I think the guy just translated from "bloodstone" but the stone in question is heliotrope which is called "heliotrop" in german

Comment (rdwz) [reply locales/de/zodiac.json:5]: > "Heliotrop" is the correct word.
> 
> I think the guy just translated from "bloodstone" but the stone in question is heliotrope which is called "heliotrop" in german

You're right, but now I know.

**Comment by rdwz** [comment]
Ohh sorry for that long delay. 
Should be fine now.

**Review by rdwz — COMMENTED** [review]
requested changes are outdated by now, everything looks fine to me. is there anything left to do on my side?

**Comment by helmasaur** [comment]
It troubles me because until someone else commented and created an other PR (#), no answers have been given…

I will still review the modifications that have been Mande on this PR (and the other one too).

**Comment by rdwz** [comment]
> It troubles me because until someone else commented and created an other PR (#49), no answers have been given and there are now two different PR about the same thing 😅.
> 
> I will still review the modifications that have been Mande on this PR (and the other one too).

The story behind this: I forgot about this pr and actually the notification of that other pr got be back on this. But I did not realize there was this other pr. I'm sorry for delay, and the confusion with now duplicate changes.

**Comment by helmasaur** [comment]
Same as https://github.com/helmasaur/zodiac-signs/pull/49#issuecomment-1691977535:

> No worries, it's not a big deal. I'm thinking about something.

**Pull request closed without merging**